mappings/net/minecraft/sortme/ILootTable.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS none/rh net/minecraft/sortme/ILootTable
```

Comment:
`IHasLootTable`? The interface doesn't implement a loot table but signalizes that the implementation *provides* one... maybe even `ILootTableProvider`? I prefer the former though.